src/flightRecorder.cpp

```diff
@@ -655,6 +685,93 @@ class Recording {
         _last_gc_id = gc_id;
     }
 
+    // Helper function to determine if a process should be included based on thresholds
+    bool shouldIncludeProcess(const ProcessInfo* info) {
+        if (info->_cpu_percent >= MIN_CPU_THRESHOLD || info->_mem_resident >= MIN_MEMORY_THRESHOLD) {
+            return true;
+        }
+
+        return false;
```

Comment:
Can be simplified to `return info->_cpu_percent >= MIN_CPU_THRESHOLD || info->_mem_resident >= MIN_MEMORY_THRESHOLD;`